.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2
+
+jobs:
+  build:
+    machine: true
+    environment:
+      CHANGE_MINIKUBE_NONE_USER: true
+    environment:
+      PYTHON: "3.6"
+      ENV_NAME: "daskernetes-test"
+    steps:
+      - checkout
+      - run:
+          command: |
+            curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.4/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+      - run:
+          command: |
+            curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+      - run:
+          command: |
+            sudo minikube start --vm-driver=none
+      - run:
+          command: |
+            sudo minikube update-context
+      - run:
+          command: |
+            JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until sudo kubectl get nodes -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1; done
+      - run:
+          command: |
+            sudo kubectl cluster-info
+      - run:
+          command: |
+            sudo kubectl run circleci-example --image=nginx
+      - run:
+          command: |
+            sudo kubectl get deployment
+      - run:
+          name: install miniconda
+          command: |
+              wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+              bash miniconda.sh -b -p $HOME/miniconda
+              export PATH="$HOME/miniconda/bin:$PATH"
+              conda config --set always_yes yes --set changeps1 no
+              sudo chown -R $USER.$USER $HOME
+      - run:
+          name: configure conda
+          command: |
+              export PATH="$HOME/miniconda/bin:$PATH"
+              conda update -q conda
+              conda env create -f ci/environment-${PYTHON}.yml --name=${ENV_NAME}
+              conda env list
+              source activate ${ENV_NAME}
+              pip install --no-deps --quiet -e .
+              conda list ${ENV_NAME}
```

Comment:
TODO: we should figure out how to cache this.  @TomAugspurger does this in the dask-ml circle-ci config.

Reply:
I've now added caching.  This replaces around one minute of conda time with twenty seconds of download-cache time.